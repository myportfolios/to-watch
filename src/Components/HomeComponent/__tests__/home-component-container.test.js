import React from "react";
import { shallow } from "enzyme";
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import {
  HomeComponentContainer,
  mapStateToProps
} from "../HomeComponentContainer";

configure({ adapter: new Adapter() });

describe("HomeComponentContainer", () => {
  let props = {
    getSuggestedMovies: jest.fn()
  };

  let wrapper = shallow(<HomeComponentContainer {...props} />);

  it("renders corectly", () => {
    expect(wrapper).toMatchSnapshot();
  });
  it("should mapStateToProps correctly", () => {
    const state = {
      movies: {
        data: [
          {
            test: "test"
          }
        ]
      }
    };
    expect(mapStateToProps(state)).toEqual({
      movies: state.movies.data
    });
  });
});
