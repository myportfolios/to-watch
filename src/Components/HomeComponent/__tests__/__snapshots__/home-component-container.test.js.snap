// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeComponentContainer renders corectly 1`] = `ShallowWrapper {}`;
